pkg/asset/manifests/openstack/cluster.go

```diff
@@ -118,7 +123,7 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 		Manifests: manifests,
 		InfrastructureRefs: []*corev1.ObjectReference{
 			{
-				APIVersion: capo.GroupVersion.String(),
+				APIVersion: capo.SchemeGroupVersion.Version,
```

Comment:
@pierreprinetti I wonder if you can help me on this one. We have this error:

```

time="2024-10-31T17:51:51Z" level=debug msg="E1031 17:51:51.908963    1925 controller.go:324] \"Reconciler error\" err=\"failed to update apiVersion in ref: failed to retrieve metadata from GVK resource: CustomResourceDefinition.apiextensions.k8s.io \\\"openstackclusters.\\\" not found\" controller=\"cluster\" controllerGroup=\"cluster.x-k8s.io\" controllerKind=\"Cluster\" Cluster=\"openshift-cluster-api-guests/y8idb02b-d2a0e-7p22w\" namespace=\"openshift-cluster-api-guests\" name=\"y8idb02b-d2a0e-7p22w\" reconcileID=\"1b7524d0-9c45-4f0c-8977-dba8a03816a9\""
```
I suspect this change is related.